zcash_client_backend/src/tor.rs

```diff
@@ -6,7 +6,7 @@ use arti_client::{config::TorClientConfigBuilder, TorClient};
 use tor_rtcompat::PreferredRuntime;
 use tracing::debug;
 
-#[cfg(feature = "lightwalletd-tonic")]
+#[cfg(feature = "tor-lightwalletd-tonic")]
```

Comment:
This change turned out to be incomplete, and cause its own compilation errors. Fixed in #1532.